spec/classes/datadog_agent_integrations_disk_spec.rb

```diff
@@ -4,42 +4,48 @@
   context 'supported agents' do
     ALL_SUPPORTED_AGENTS.each do |agent_major_version|
       let(:pre_condition) { "class {'::datadog_agent': agent_major_version => #{agent_major_version}}" }
+
       if agent_major_version == 5
-        let(:conf_file) { "/etc/dd-agent/conf.d/disk.yaml" }
+        let(:conf_file) { '/etc/dd-agent/conf.d/disk.yaml' }
       else
         let(:conf_file) { "#{CONF_DIR}/disk.d/conf.yaml" }
       end
 
       it { is_expected.to compile.with_all_deps }
-      it { is_expected.to  contain_file(conf_file).with_content(
-        %r{\s+use_mount:\s+no[\r]*$}
-      ).with(
-        owner: DD_USER,
-        group: DD_GROUP,
-        mode: PERMISSIONS_PROTECTED_FILE,
-      )}
+      it {
+        is_expected.to contain_file(conf_file).with_content(
+          %r{\s+use_mount:\s+no[\r]*$},
+        ).with(
+          owner: DD_USER,
+          group: DD_GROUP,
+          mode: PERMISSIONS_PROTECTED_FILE,
+        )
+      }
       it { is_expected.to contain_file(conf_file).that_requires("Package[#{PACKAGE_NAME}]") }
       it { is_expected.to contain_file(conf_file).that_notifies("Service[#{SERVICE_NAME}]") }
 
       context 'compile errors for incorrect values' do
-        let(:params) {{ use_mount: 'heaps' }}
+        let(:params) { { use_mount: 'heaps' } }
```

Comment:
Since rubcop doesn't enforce this (it accepts both syntaxes), I think it can stay like that. I don't want to enforce syntax consistency by hand (since future PRs could be re-adding this syntax anyway).